diff --git forkSrcPrefix/app/src/util/needSubscribe.ts forkDstPrefix/app/src/util/needSubscribe.ts
index 6ba7848..b0427bf 100644
--- forkSrcPrefix/app/src/util/needSubscribe.ts
+++ forkDstPrefix/app/src/util/needSubscribe.ts
@@ -2,22 +2,24 @@ import {showMessage} from "../dialog/message";
 import {getCloudURL} from "../config/util/about";
 
 export const needSubscribe = (tip = window.siyuan.languages._kernel[29]) => {
-    if (window.siyuan.user && (window.siyuan.user.userSiYuanProExpireTime === -1 || window.siyuan.user.userSiYuanProExpireTime > 0)) {
-        return false;
-    }
-    if (tip) {
-        if (tip === window.siyuan.languages._kernel[29] && window.siyuan.config.system.container === "ios") {
-            showMessage(window.siyuan.languages._kernel[122]);
-        } else {
-            if (tip === window.siyuan.languages._kernel[29]) {
-                tip = window.siyuan.languages._kernel[29].replace("${url}", getCloudURL("subscribe/siyuan"));
-            }
-            showMessage(tip);
-        }
-    }
-    return true;
+    return false;
+    // if (window.siyuan.user && (window.siyuan.user.userSiYuanProExpireTime === -1 || window.siyuan.user.userSiYuanProExpireTime > 0)) {
+    //     return false;
+    // }
+    // if (tip) {
+    //     if (tip === window.siyuan.languages._kernel[29] && window.siyuan.config.system.container === "ios") {
+    //         showMessage(window.siyuan.languages._kernel[122]);
+    //     } else {
+    //         if (tip === window.siyuan.languages._kernel[29]) {
+    //             tip = window.siyuan.languages._kernel[29].replace("${url}", getCloudURL("subscribe/siyuan"));
+    //         }
+    //         showMessage(tip);
+    //     }
+    // }
+    // return true;
 };
 
 export const isPaidUser = () => {
-    return window.siyuan.user && (0 === window.siyuan.user.userSiYuanSubscriptionStatus || 1 === window.siyuan.user.userSiYuanOneTimePayStatus);
+    return true;
+    // return window.siyuan.user && (0 === window.siyuan.user.userSiYuanSubscriptionStatus || 1 === window.siyuan.user.userSiYuanOneTimePayStatus);
 };


diff --git forkSrcPrefix/kernel/model/conf.go forkDstPrefix/kernel/model/conf.go
index 3c3add3..1502b5b 100644
--- forkSrcPrefix/kernel/model/conf.go
+++ forkDstPrefix/kernel/model/conf.go
@@ -836,22 +836,24 @@ func InitBoxes() {
 }
 
 func IsSubscriber() bool {
-	u := Conf.GetUser()
-	return nil != u && (-1 == u.UserSiYuanProExpireTime || 0 < u.UserSiYuanProExpireTime) && 0 == u.UserSiYuanSubscriptionStatus
+	// u := Conf.GetUser()
+	// return nil != u && (-1 == u.UserSiYuanProExpireTime || 0 < u.UserSiYuanProExpireTime) && 0 == u.UserSiYuanSubscriptionStatus
+	return true
 }
 
 func IsPaidUser() bool {
 	// S3/WebDAV data sync and backup are available for a fee https://github.com/siyuan-note/siyuan/issues/8780
 
-	if IsSubscriber() {
-		return true
-	}
+	// if IsSubscriber() {
+	// 	return true
+	// }
 
-	u := Conf.GetUser()
-	if nil == u {
-		return false
-	}
-	return 1 == u.UserSiYuanOneTimePayStatus
+	// u := Conf.GetUser()
+	// if nil == u {
+	// 	return false
+	// }
+	// return 1 == u.UserSiYuanOneTimePayStatus
+	return true
 }
 
 const (
